.github/workflows/license_check.yml

```diff
@@ -0,0 +1,18 @@
+name: Check license of dependencies
+on: [push, pull_request]
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: WillAbides/setup-go-faster@v1.7.0
+      with:      
+        go-version: '1.18beta2'
+
+    - name: install lian
+      run: go install lucor.dev/lian@latest
```

Comment:
Development versions in production? I’m not so sure about that. Can’t we stick to a stable release?

Reply:
the tool requires go 1.18 so it is needed if we want to install via `go install`. 
We could install using the binary but in this way we need to use a specific version. (the tool works but it is into an early stage so an update could be required soon :) )
If the latest is ok, we could also integrate into the static check workflow instead of have a dedicated one.

Reply:
> the tool requires go 1.18 so it is needed if we want to install via `go install`.

We should not depend upon Go 1.18 before it is released IMHO.

Reply:
I agree with @andydotxyz  about that. What is the reason for the 1.18 requirement after all? Not everyone will have Go 1.18 right as it is released, so it makes it a little harder to run locally for some. Is there any chance that the requirement could be lowered? If not, I think we sadly need to wait a few weeks for the stable release.

Reply:
The tool uses [buildinfo.ReadFile](https://pkg.go.dev/debug/buildinfo@go1.18rc1#ReadFile) that is available starting from 1.18, hence the dependency at compile time.
We'd install from binary and update later to use `go install` but since go 1.18, as @Jacalz said, should be available in a few weeks I'd say it is ok to wait and update the PR once 1.18 is released. WDYT?

Reply:
Ah, I see. Sounds great to me 👍

Reply:
This should be unblocked now that Go 1.18 finally has been released :)

Reply:
Done in [881be11](https://github.com/fyne-io/fyne/pull/2773/commits/881be112a5e16f505b6c614eb8d73cc8f0a08b3d)